tools/testScripts/testLibrary.shlib

```diff
@@ -1687,5 +1687,6 @@ function setBucket {
 
 	local port=${CLUSTER_NAME_PORT_MAP[$cluster]:-}
 	echo "$CURL -X POST -u $DEFAULT_ADMIN:$DEFAULT_PW http://127.0.0.1:$port/pools/default/buckets/$bucketName -d $setting=$value"
-	$CURL -X POST -u $DEFAULT_ADMIN:$DEFAULT_PW http://127.0.0.1:$port/pools/default/buckets/$bucketName -d $setting=$value
+	output=$($CURL -X POST -u $DEFAULT_ADMIN:$DEFAULT_PW http://127.0.0.1:$port/pools/default/buckets/$bucketName -d $setting=$value)
```

Comment:
Why do you want to change the output here?

Reply:
No change to output. Without my change here, any error will be displayed without a return. The next output in the test will continue on the same line and it is confusing.